src/InfRcTransport.cc

```diff
@@ -207,6 +207,10 @@ InfRcTransport::InfRcTransport(Context* context, const ServiceLocator *sl,
         try {
             ibPhysicalPort = sl->getOption<int>("devport");
         } catch (ServiceLocator::NoSuchKeyException& e) {}
+    } else {
+        // HACK! This is just to fix the broken test client!  Don't merge this nonsense!
+        ibDeviceName   = "rocep3s0f0v0";
```

Comment:
Do we know yet what the difference btw rc test client and nfapp's ibDeviceName will be? It's ok if we don't know that yet, just bringing it to your attention